internal/gouging/gouging.go

```diff
@@ -0,0 +1,117 @@
+package gouging
+
+import (
+	"fmt"
+
+	"gitlab.com/NebulousLabs/errors"
+	"go.sia.tech/renterd/internal/utils"
+	rhpv2 "go.sia.tech/renterd/rhp/v2"
+	"go.sia.tech/siad/modules"
+	"go.sia.tech/siad/types"
+)
+
+var (
+	// ErrGougingDetected is returned when price gouging was detected in one of
+	// the fields of either the price table or the host's external settings.
+	ErrGougingDetected = errors.New("price gouging detected")
```

Comment:
do we need this? Can't the caller assume that if a gouging check returns an error, it's because it detected gouging?

Reply:
I'll remove it, I sometimes do this when I return multiple errors that belong to the same "category". By have a common error that's exposed the caller can do `errors.Is` checks. It's not being used yet so I agree that was a bit premature